packages/blockfrost/test/blockfrostProvider.test.ts

```diff
@@ -409,4 +409,43 @@ describe('blockfrostProvider', () => {
       slot: 37_767_194
     });
   });
+
+  test('rewardsHistory', async () => {
+    const pool_id = 'pool1pu5jlj4q9w9jlxeu370a3c9myx47md5j5m2str0naunn2q3lkdy';
+    const mockedResponse = [
```

Comment:
This test isn't covering the multiple page scenario. See: https://github.com/input-output-hk/cardano-js-sdk/blob/02f73783e39c7cb2c0e56d675bd2c5a3ae175903/packages/blockfrost/test/blockfrostProvider.test.ts#L70-L101